client/src/pages/EditProject.js

```diff
@@ -0,0 +1,327 @@
+              multiline
+              rows="5"
+              classes={{ root: classes.formLine }}
+              value={description}
+              fullWidth={true}
+              onChange={this.handleInput}
+              type="description"
+              variant="outlined"
+            />
+
+            <OutlinedSelect
+              useLabel
+              name="industry"
+              labelText="Industry"
+              selectId="industry"
+              setState={this.handleInput}
+              selectName="industry"
+              value={industry}
+            >
+              {
+                industries.map(industry => {
+                  return (
+                    <option
+                      key={industry.id}
+                      value={industry.name}
+                    >
+                      {industry.name}
+                    </option>
+                  )
+                })
+              }
+            </OutlinedSelect>
+            <OutlinedSelect
+              useLabel
+              name="location"
+              labelText="Location"
+              selectId="location"
+              setState={this.handleInput}
+              selectName="location"
+              value={location}
+            >
+              {
+                locations.map(location => {
+                  return (
+                    <option
+                      key={location.id}
+                      value={location.name}
+                    >
+                      {location.name}
+                    </option>
+                  )
+                })
+              }
+            </OutlinedSelect>
+            <UploadImages setImages={this.setImages} showMany={true} images={updatedImages} deleteImage={this.deleteImage} />
+            <Typography variant="h4">Funding Goal Amount</Typography>
+            <OutlinedInput
+              name="fundingGoal"
+              id="fundingGoal"
+              value={fundingGoal}
+              fullWidth={true}
+              onChange={this.handleInput}
+              type="fundingGoal"
+              variant="outlined"
+              required
+              error={validation.fundingGoal.isInvalid}
+              helpertext={validation.fundingGoal.message}
+              startAdornment={<InputAdornment position="start">$</InputAdornment>}
+            />
+            <Typography variant="h4">Funding Deadline</Typography>
+            <TextField
+              className={classes.dateField}
+              name="fundingDeadline"
+              id="fundingDeadline"
+              type="date"
+              value={fundingDeadline}
+              onChange={this.handleInput}
+              required
+              InputLabelProps={{
+                shrink: true,
+              }}
+            />
+            <Button classes={{ root: classes.button }} type="submit" variant="contained" color="primary" disabled={this.disableSubmit()}>
+              Submit
+            </Button>
```

Comment:
I would center the button, but not blocking of course

Reply:
Button has been centered now